package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "medic-couch2pg",
-  "version": "3.3.0",
-  "description": "replicate medic data to postgres database",
+  "name": "cht-couch2pg",
+  "version": "3.4.0",
```

Comment:
Are we going to do an immediate release? I would like to address first #82 , I'm trying to prioritize it but maybe it's better in the meantime to move the version change in a new PR once we decide to release.

Reply:
I think we can release immediately. 
Since it will be a different npm package, we're not worried someone might upgrade accidentally.